riscv/encoding.h

```diff
@@ -4,7 +4,7 @@
 
 /*
  * This file is auto-generated by running 'make' in
- * https://github.com/riscv/riscv-opcodes (02b4866)
+ * https://github.com/riscv/riscv-opcodes (b6fea6c)
```

Comment:
Please update the URL here if this was not generated from the official riscv-opcodes repo.

Reply:
Done, thanks for noticing. It's a little strange to edit an auto-generated file, but it makes sense for now.